airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -88,12 +88,12 @@
 - destinationDefinitionId: 8aaf41d0-f6d2-46de-9e79-c9540f828142
   name: Keen
   dockerRepository: airbyte/destination-keen
-  dockerImageTag: 0.1.0
+  dockerImageTag: 0.2.0
```

Comment:
I've bumped versions here and in the `<uuid>.json` file, however I don't see updated version number in settings -> Destination. Should I do something more ?

Reply:
instances don't autoupdate versions for stability reasons. The user must update it by hand. Versions specified in this/the JSON files are used the very first time Airbyte runs.